DOL.WHD.Section14c.Web/karma.conf.js

```diff
@@ -4,7 +4,7 @@
 var IS_DEBUG = process.env.DEBUG || false;
 var BROWSERS = [IS_DEBUG ? 'Chrome' : 'PhantomJS'];
 
-var webpackConfig = require('./webpack.test.config');
+var webpackConfig = require('./webpack.config');
```

Comment:
My only concern is that the test config includes coverage instrumentation.  Tinkering around with it locally, I see that instrumentation causes a bunch of linting errors, though.  I'm kind of at a loss about how to deal with that.